astropy/wcs/wcs.py

```diff
@@ -519,10 +519,12 @@ def __init__(
         fix=True,
         translate_units="",
         _do_set=True,
-        preserve_units=False,
+        preserve_units=None,
```

Comment:
I still don't get what `None` is doing here and why it is not listed as an option in the dicstring. I think you want it to be `False` by default.

Reply:
Oops another leftover, not sure how I missed that